app-misc/toilet/toilet-0.3~git.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Free replacement for the FIGlet utility"
+DESCRIPTION="Free replacement for the FIGlet utility \
+- a program for making large letters out of ordinary text."
+HOMEPAGE="https://github.com/cacalabs/toilet"
+COPYRIGHT="2004 Sam Hocevar"
+LICENSE="WTFPL"
+REVISION="1"
+srcGitRev=3eb9d58037afb0a1baec6dca82caa045fb2217c0
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="toilet-0.3~$srcGitRev.tar.gz"
+SOURCE_DIR="toilet-$srcGitRev"
+
+ARCHITECTURES="?x86 ?x86_gcc2 x86_64"
```

Comment:
We like to have it with another order:
```
ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```
(most of our recipes use this order.)
Thanks :)